internal/compiler/filesparser.go

```diff
@@ -51,6 +50,10 @@ func (t *parseTask) Path() tspath.Path {
 	return t.path
 }
 
+func (t *parseTask) isRoot() bool {
+	return t.includeReason.kind == fileIncludeKindRootFile || t.includeReason.kind == fileIncludeKindLibFile
```

Comment:
lib files are always root? We previously only set true in `addRootTask`.

Reply:
Yes those are lib files - LibReference is different include kind